wies/core/jinja2/parts/sidepanel.html

```diff
@@ -0,0 +1,45 @@
+<div class="rvo-dialog__background" id="sidepanel-overlay">
+  <dialog class="rvo-dialog rvo-dialog--wit rvo-dialog--inset-inline-end"
+          aria-expanded="true"
+          aria-label="{{ panel_title }}"
+          id="sidepanel"
+          open>
+    
+    <div class="rvo-dialog__close">
+      <button class="utrecht-button utrecht-button--subtle rvo-dialog__close-button utrecht-button--rvo-md"
+              type="button"
+              onclick="window.location.href='{{ close_url|safe }}'">
```

Comment:
is it really necesary to do this here? marking safe means that we know for sure that `close_url` never contains untrusted user generated code. you can also do it the other way around: mark it as safe in the view. that way you explicitly mark what's safe: https://docs.djangoproject.com/en/6.0/ref/utils/#django.utils.safestring.mark_safe